### ISSUE TITLE
Camera Based Outline, Geometry with Sharp Edges

### ISSUE THREAD

**Issue by tylo** [adrian-miasik/unity-shaders#7]
Have you tried testing your Camera Based Outline shader on a primitive cube? It works well on a sphere, but on the cube, the vertex expansion along the normals cause the sides of the cubes to separate from one another.

See attached picture.
![image](https://user-images.githubusercontent.com/173893/82846276-01397280-9eb6-11ea-9052-beb974241196.png)

**Comment by tylo** [comment]
I suppose the cube itself could be to blame? That each side has their own vertices, instead of shared vertices?

**Comment by jonathanperret** [comment]
Yes, for this kind of outline shader you will get much better results with normals that are shared between faces.
